src/main/java/me/totalfreedom/totalfreedommod/rank/RankManager.java

```diff
@@ -61,6 +61,11 @@ public Displayable getDisplay(CommandSender sender)
         {
             return Title.EXECUTIVE;
         }
+        
+        if (ConfigEntry.SERVER_ASSTISTANT_EXECUTIVES.getList().contains(player.getName()) && plugin.al.isAdmin(player))
```

Comment:
This line here, the config entry has assistant spelt incorrectly, the suggestion is to refactor it to be correctly spelt.